warcio/archiveiterator.py

```diff
@@ -41,7 +41,8 @@ class ArchiveIterator(six.Iterator):
 
     def __init__(self, fileobj, no_record_parse=False,
                  verify_http=False, arc2warc=False,
-                 ensure_http_headers=False, block_size=BUFF_SIZE):
+                 ensure_http_headers=False, block_size=BUFF_SIZE,
+                 check_digests=True):  # XXX is this the right default for check_digests ?
```

Comment:
No, I think it should be off by default.

Trying to follow the Postel's Law by default, "be strict about writing WARC records, lax about reading them" :)